cfme/tests/infrastructure/test_vm_reconfigure.py

```diff
@@ -191,3 +191,303 @@ def test_vm_reconfig_add_remove_hw_hot(provider, small_vm, ensure_vm_running, ch
+    """
+    Polarion:
+        assignee: nansari
+        initialEstimate: 1/6h
+        caseimportance: medium
+        caselevel: integration
+        caseposneg: positive
+        caseautomation: Non Automated
+        testtype: functional
+        startsin: 5.6
+        casecomponent: infra
+        testSteps:
+            1. get new configured appliance ->add vmware provider
+            2. provision 2 new vms
+            3. power off 1 vm -> select both vms
+            4. configure-->reconfigure vm
+            5. power on vm
+            6. check changes
+    """
+    pass
+
+
+@pytest.mark.manual
+@pytest.mark.tier(1)
+def test_vm_reconfig_attach_iso_vsphere67_nested():
+    """
+
+    Bugzillas:
+        * 1533728
+
+    Polarion:
+        assignee: nansari
+        initialEstimate: 1/6h
+        caseimportance: high
+        caselevel: integration
+        caseposneg: positive
+        caseautomation: Non Automated
+        testtype: functional
+        startsin: 5.10
+        casecomponent: infra
+        testSteps:
+            1. Add vmware provider
+            2. provision 1 new vms
+            3. Run a Smartstate analysis on the Datastore (to get the list of ISO files)
+            4. configure-->reconfigure vm
+            5. Select an ISO (This should allow you to select an ISO to attach to the CD drive)
+            6. check changes
+    """
+    pass
+
+
+@pytest.mark.manual
+@pytest.mark.tier(2)
+def test_reconfigure_vm_vmware_sockets_multiple():
+    """ Test changing the cpu sockets of multiple vms at the same time.
+
+    Polarion:
+        assignee: nansari
+        initialEstimate: 1/6h
+        caseimportance: medium
+        caselevel: integration
+        caseposneg: positive
+        caseautomation: Non Automated
+        testtype: functional
+        startsin: 5.6
+        casecomponent: infra
+        testSteps:
+            1. get new configured appliance ->add vmware provider
+            2. provision 2 new vms
+            3. power off 1 vm -> select both vms
+            4. configure-->reconfigure vm
+            5. increase/decrease counts
+            6. power on vm
+            7. check changes
+    """
+    pass
+
+
+@pytest.mark.manual
+@pytest.mark.tier(2)
+def test_reconfigure_vm_vmware_cores_multiple():
+    """ Test changing the cpu cores of multiple vms at the same time.
+
+    Polarion:
+        assignee: nansari
+        initialEstimate: 1/6h
+        caseimportance: medium
+        caselevel: integration
+        caseposneg: positive
+        caseautomation: Non Automated
+        testtype: functional
+        startsin: 5.6
+        casecomponent: infra
+        testSteps:
+            1. get new configured appliance ->add vmware provider
+            2. provision 2 new vms
+            3. power off 1 vm -> select both vms
+            4. configure-->reconfigure vm
+            5. increase/decrease counts
+            6. power on vm
+            7. check changes
```

Comment:
I think you should keep the original test steps and expected results somewhere, otherwise it's not clear what to expect from this changes

```
        testSteps:	
            1. Hot increase	
            2. Hot Decrease	
            3. Cold Increase	
            4. Cold Decrease	
            5. Hot + Cold Increase	
            6. Hot + Cold Decrease	
        expectedResults:	
            1. Action should fail	
            2. Action should fail	
            3. Action should succeed	
            4. Action should succeed	
            5. Action should fail	
            6. Action should Error
```